Lib/datetime.py

```diff
@@ -732,6 +732,15 @@ def fromordinal(cls, n):
         y, m, d = _ord2ymd(n)
         return cls(y, m, d)
 
+    @classmethod
+    def fromisoformat(cls, date_string):
+        """Constructs a date from an RFC 3339 string, a strict subset of ISO 8601.
```

Comment:
Convention is to use imperative form ("Construct"). This comment applies in other places.